src/python/WMCore/WMStats/CherryPyThreads/CleanUpTask.py

```diff
@@ -41,13 +41,16 @@ def cleanUpArchivedRequests(self, config):
         self.logger.info("archived list %s", requestNames)
 
         for req in requestNames:
-            self.logger.info("deleting %s data", req)
+            self.logger.info("Deleting data for: %s", req)
             try:
                 result = self.wmstatsDB.deleteDocsByWorkflow(req)
             except Exception as ex:
                 self.logger.error("deleting %s failed: %s", req, str(ex))
                 for line in traceback.format_exc().rstrip().split("\n"):
                     self.logger.error(" " + line)
             else:
-                self.logger.info("%s deleted", len(result))
+                if result is None:
+                    self.logger.info("there were no documents to delete.")
```

Comment:
I think it should return some thing other than None if the call was successful. (Maybe should put warning message instead?)

Reply:
I don't have much info on why it happened, but I think the reason it returned null is that there were actually no documents in wmstats to be deleted, which then returns None from this line:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/Database/CMSCouch.py#L272

Reply:
I see I think you are right, Alan. as always. 😄